app/scripts/models/auth_brokers/first-run.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The auth broker to coordinate authenticating for Sync when
+ * embedded in the Firefox firstrun flow.
+ */
+
+define([
+  'models/auth_brokers/fx-desktop-v2'
+], function (FxDesktopV2AuthenticationBroker) {
+  'use strict';
+
+  var FirstRunAuthenticationBroker = FxDesktopV2AuthenticationBroker.extend({
+    haltAfterResetPasswordConfirmationPoll: false,
```

Comment:
Another approach I tried, instead of creating a complete new broker, is to pass these values as configuration to the FxDesktopV2AuthenticationBroker. I settled on creating the FirstRunAuthenticationBroker because I need it for #2705